crates/setup-tool/src/main.rs

```diff
@@ -136,7 +136,6 @@ fn run() -> Result<()> {
     let rpc_listen_address = args.value_of(RPC_LISTEN_ADDRESS).unwrap();
     let lumos_callback_address = args.value_of(LUMOS_CALLBACK_ADDRESS).unwrap();
     let lumos_endpoint = args.value_of(LUMOS_ENDPOINT).unwrap();
-    let rollup_contract_path = args.value_of(ROLLUP_CONTRACT).unwrap();
```

Comment:
Why?

Reply:
Oops, I added some new config options below here and considered unnecessary so far. While  I remove those codes and delete this line by accident. Gonna fix it.